src/Lykke.SlackNotifications/SlackNotificationsSender.cs

```diff
@@ -23,14 +24,13 @@ public SlackNotificationsSender(IMessageProducer<SlackMessageQueueEntity> queue,
             _ownQueue = ownQueue;
         }
 
-
         public async Task SendAsync(string type, string sender, string message)
         {
             var slackMessage = new SlackMessageQueueEntity
             {
                 Type = type,
                 Message = message,
-                Sender = sender
+                Sender = $"[{DateTime.UtcNow.ToString(_timestampFormat)}] {sender}",
```

Comment:
exactly you need to add to the sender?
Maybe better in the message

Reply:
If we add a timestamp at the beginning of the message then it is placed in the middle of resulting slack message. This is not convenient.